### DISCUSSION TITLE
接收地址变了，出了1.5个币找不到了？

### DISCUSSION THREAD

**Discussion by yangyang8921** [Chia-Network/chia-blockchain#4677] [Q&A]
钱包接收地址不知道是自动还是无意更新了，但是耕种地址没有改，暴了两个0.75，币没到账，区块浏览器能查到新地址有1.5个币，但是钱包找不到，求解怎么找回？
另外一直有疑问，接收地址变得话是不是就不会爆块了，耕种地址改成和接收一样就可以了么，这两个地址必须保持一直么？

**Comment by HaitaoMiao** [comment]
不是2个币一快爆吗? 你怎么是1.5呢

**Comment by 7by24** [comment]
接的矿池？

**Comment by yangyang8921** [comment]
两次爆快，一次0.75



在 2021-05-13 11:07:02，"HaitaoMiao" ***@***.***> 写道：

不是2个币一快爆吗? 你怎么是1.5呢

**Comment by yangyang8921** [comment]
从来没接过



在 2021-05-13 12:21:48，"elarkasi" ***@***.***> 写道：

接的矿池？

**Comment by eugenetsou** [comment]
我也發生一樣的狀況... #
